src/scss/_status.scss

```
@@ -0,0 +1,77 @@
+/// @access private
+/// @output styles for state animation for the box-styled radio inputs
+@mixin _oFormsStatus {
+	.o-forms__status {
+		@include oTypographySize($scale: -1, $line-height: 1);
+		color: inherit;
+		display: block;
+
+		&:before,
+		&:after {
+			content: '';
+			display: inline-block;
+			vertical-align: middle;
+		}
+
+		&.o-forms__status--icon-only:after {
+			@include oNormaliseVisuallyHidden;
+		}
+	}
+
+	.o-forms--saving .o-forms__status {
+		&:after {
+			content: 'Saving';
+		}
+
+		&:before {
+			@include oLoadingContent($opts: ('color': 'dark', 'size': 'mini'));
```

The loading icon isn't showing because `color` is `theme` now.

---

Oh wait I got this 😁

```suggestion
			@include oLoadingContent($opts: ('theme': 'dark', 'size': 'mini'));
```